app.py

```diff
@@ -602,6 +600,21 @@ def narrative_status():
     return(flask.Response(json.dumps(resp_doc), 200, mimetype='application/json'))
 
 
+@app.route("/reaper/", methods=['GET'])
+def reaper_endpoint():
+    """
+    Endpoint that just runs the reaper once and returns the status
+    """
+    logger.info({"message": "Reaper endpoint called"})
+    try:
+        reaper()
+        resp = flask.Response("Reaper success")
```

Comment:
Could we have the endpoint return the number of containers reaped, since reaper() now returns that info?